x-pack/test/detection_engine_api_integration/utils/delete_all_signals.ts

```diff
@@ -7,25 +7,37 @@
 
 import type SuperTest from 'supertest';
 import type { ToolingLog } from '@kbn/tooling-log';
+import type { Client } from '@elastic/elasticsearch';
 import { DETECTION_ENGINE_INDEX_URL } from '@kbn/security-solution-plugin/common/constants';
 import { countDownTest } from './count_down_test';
 
 /**
- * Deletes the signals index for use inside of afterEach blocks of tests
- * @param supertest The supertest client library
+ * Deletes all alerts from a given index or indices, defaults to `.alerts-security.alerts-default`
+ * For use inside of afterEach blocks of tests
  */
-export const deleteSignalsIndex = async (
+export const deleteAllSignals = async (
   supertest: SuperTest.SuperTest<SuperTest.Test>,
-  log: ToolingLog
+  log: ToolingLog,
+  es: Client,
+  index: string[] = ['.alerts-security.alerts-default']
```

Comment:
I think since we are seemingly only using the `.alerts` index or the `.preview.alerts` index it might be good to update the type for the `index` parameter accepted by this function to be a bit more strict. Also, I feel since there isn't an api available to get the full alerts index name with the space included, we should either update the `index` parameter to remove the spaceId at the end of the index name (`-default`) and replace it with the `-*` char [1] or allow the spaces service to be passed in from the FTR test and run a `deleteByQuery` on all individual alerts indices that could potentially be created [2].


[1]: 
```suggestion
  log: ToolingLog,
  es: Client,
  index:
    | '.alerts-security.alerts-*'
    | '.preview.alerts-security.alerts-*' = '.alerts-security.alerts-*'
```

[2]: 
```suggestion
  log: ToolingLog,
  es: Client,
  spaces: {
    create: (space: any) => Promise<void>;
    delete: (spaceId: string) => Promise<void>;
    getAll: () => Promise<Space[]>;
  },
  index:
    | '.alerts-security.alerts-'
    | '.preview.alerts-security.alerts-' = '.alerts-security.alerts-'
```